components/elements/buttons/ButtonLink/ButtonLink.test.ts

```diff
@@ -1,5 +1,5 @@
 import ButtonLink from './ButtonLink.vue';
-import { stubbedRender } from '@/testUtils';
+import { stubbedRender } from '~/testUtils';
 
 describe('<ButtonLink />', () => {
   const defaultAttrs = { class: 'custom-link' };
```

Comment:
Test "renders with the passed link" fails on my end. For some reason, it's not looking at the `defaultProps` that you passed in, and still wants to match with what's defined in `ButtonLink.ts`.

Reply:
Fixed the disappearing button issue. Also confirm that 
![image](https://user-images.githubusercontent.com/27028426/142065418-0dc53fe6-ec41-43ad-94ad-df49eaaa67f3.png)
the site name isn't on the mobile navbar

Reply:
Sweet. I've also fixed the failing test in https://github.com/leaderboardsgg/leaderboard-site/pull/130/commits/9e8ea95baf76a8fc7d7d06d8f77a7f326a3c20ca